src/internal/mod.rs

```diff
@@ -417,60 +417,6 @@ impl From<DeviceAddResult> for DeviceContext {
     }
 }
 
-#[derive(Clone, Debug, Eq, Hash, PartialEq)]
-pub struct DeviceAddResult {
```

Comment:
This was moved to the `user_api` module as it's only ever used with user operations